ols/src/query_helpers/chat_history.py

```diff
@@ -0,0 +1,44 @@
+"""Produce chat history object using list of Conversations."""
+
+from typing import List
+
+from langchain.schema import ChatMessage as langchainCM
+from llama_index.llms import ChatMessage, MessageRole
+
+from ols.src.cache.conversation import Conversation
+
+
+def get_llama_index_chat_history(
+    conversations: List[Conversation],
+) -> List[ChatMessage]:
+    """Produce list of ChatMessage object using list of Conversations for llama_index."""
+    chat_history: List[Conversation] = []
+    if conversations is None or len(conversations) < 1:
```

Comment:
can it be `None`?

Reply:
I dont expect to be None , but even if we get a None we will return a empty History. You okay with that @tisnik

Reply:
I meant, if it can be None, the type of parameter should be `Optional[...]`.

Reply:
made it `list[Conversation] | None`

Reply:
I don't know if something changed, but we should always use the `Optional` instead of `something | None`.

Reply:
> I dont expect to be None , but even if we get a None we will return a empty History

seems like a weird way to define what ends up being a constructor.  I think you should just not accept None/empty lists.